### ISSUE TITLE
Failing tests for clickhouse-server:22.9-alpine

### ISSUE THREAD

**Issue by elpikel** [plausible/analytics#3694]
### Past Issues Searched

- [X] I have searched open and closed issues to make sure that the bug has not yet been reported

### Issue is a Bug Report

- [X] This is a bug report and not a feature request, nor asking for self-hosted support

### Using official Plausible Cloud hosting or self-hosting?

Self-hosting

### Describe the bug

I was following [CONTRIBUTING.md](https://github.com/plausible/analytics/blob/master/CONTRIBUTING.md) and I encounter following errors when running `mix test`:

```elixir
  1) test collapse order across and within parts (Plausible.Session.CacheStoreTest)
     test/plausible/session/cache_store_test.exs:163
     ** (Ch.Error) Code: 115. DB::Exception: Unknown setting final. (UNKNOWN_SETTING) (version 22.9.7.34 (official build))
     
     code: session = Plausible.ClickhouseRepo.one!(session_q, settings: [final: true])
     stacktrace:
       (ecto_sql 3.11.1) lib/ecto/adapters/sql.ex:1054: Ecto.Adapters.SQL.raise_sql_call_error/1
       (ecto_ch 0.3.2) lib/ecto/adapters/clickhouse.ex:309: Ecto.Adapters.ClickHouse.execute/5
       (ecto 3.11.1) lib/ecto/repo/queryable.ex:232: Ecto.Repo.Queryable.execute/4
       (ecto 3.11.1) lib/ecto/repo/queryable.ex:19: Ecto.Repo.Queryable.all/3
       (ecto 3.11.1) lib/ecto/repo/queryable.ex:162: Ecto.Repo.Queryable.one!/3
       test/plausible/session/cache_store_test.exs:177: (test)

.

  2) test collapse order across parts (Plausible.Session.CacheStoreTest)
     test/plausible/session/cache_store_test.exs:133
     ** (Ch.Error) Code: 115. DB::Exception: Unknown setting final. (UNKNOWN_SETTING) (version 22.9.7.34 (official build))
     
     code: session = Plausible.ClickhouseRepo.one!(session_q, settings: [final: true])
     stacktrace:
       (ecto_sql 3.11.1) lib/ecto/adapters/sql.ex:1054: Ecto.Adapters.SQL.raise_sql_call_error/1
       (ecto_ch 0.3.2) lib/ecto/adapters/clickhouse.ex:309: Ecto.Adapters.ClickHouse.execute/5
       (ecto 3.11.1) lib/ecto/repo/queryable.ex:232: Ecto.Repo.Queryable.execute/4
       (ecto 3.11.1) lib/ecto/repo/queryable.ex:19: Ecto.Repo.Queryable.all/3
       (ecto 3.11.1) lib/ecto/repo/queryable.ex:162: Ecto.Repo.Queryable.one!/3
       test/plausible/session/cache_store_test.exs:140: (test)
```

This can be fixed with new version of `clickhouse`: `clickhouse/clickhouse-server:23.12-alpine` in `Makefile` or `clickhouse-prod` version can be used.

I can open PR if we agree on solution.

### Expected behavior

All tests should pass when following [CONTRIBUTING.md](https://github.com/plausible/analytics/blob/master/CONTRIBUTING.md).

### Environment

```markdown
- OS: macOS Sonoma 14.2.1
- Browser: N/A
- Browser Version: N/A
```

**Comment by ruslandoga** [comment]
👋 @elpikel 

Maybe `clickhouse` rule can use `latest` tag similarly to the [`postgres` rule?](https://github.com/plausible/analytics/blob/cba5f55ba1ac803c040109bbc21ac8b7f7f3c1b2/Makefile#L31-L32)

Please do open a PR :)

**Comment by elpikel** [comment]
> 👋 @elpikel
> 
> Maybe `clickhouse` rule can use the `latest` tag similarly to the [`postgres` rule?](https://github.com/plausible/analytics/blob/cba5f55ba1ac803c040109bbc21ac8b7f7f3c1b2/Makefile#L31-L32)
> 
> Please do open a PR with the approach you think is best and we'll continue the discussion there :)

Please check: https://github.com/plausible/analytics/pull/3701

**Comment by ruslandoga** [comment]
Closed in #3701

Thank you @elpikel!
